tests/providers/gcp/gcp_test.py

```diff
@@ -824,6 +825,26 @@ def testBlockOperation(self, mock_gcb_api):
       FAKE_GCB.BlockOperation(MOCK_GCB_BUILDS_CREATE)
 
 
+class GoogleKubernetesEngineTest(unittest.TestCase):
+  """Test Google Kubernetes Engine class."""
+  # pylint: disable=line-too-long
+
+  FAKE_GKE = gke.GoogleKubernetesEngine('fake-target-project')
+  MOCK_GKE_CLUSTER_OBJECT = {
+      "name": "test-cluster",
+      "location": "fake-region"
+  }
+
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.gke.GoogleKubernetesEngine.GkeApi')
+  def testGetCluster(self, mock_gke_api):
+    """Test GKE cluster Get operation."""
+    api_cluster_object = mock_gke_api.return_value.projects.return_value.locations.return_value.clusters.return_value
+    api_cluster_object.get.return_value.execute.return_value = GoogleKubernetesEngineTest.MOCK_GKE_CLUSTER_OBJECT
+    get_results = GoogleKubernetesEngineTest.FAKE_GKE.GetCluster('projects/fake-project/locations/fake-region/clusters/fake-cluster')
+    self.assertEqual(GoogleKubernetesEngineTest.MOCK_GKE_CLUSTER_OBJECT, get_results)
```

Comment:
I'm fine with the first two statements here be left on one line, but the last two can be broken down into many shorter lines easily enough. Can you please move the `pylint` statement to here to scope it down a bit?

Reply:
moved the lint statement before the mock.patch line to avoid complains, when I break mock.patch lines they don't look necessarily readable for me, but I can break it if you think it looks better, I can also do the same for all other tests.

Reply:
Yeah that's exactly what I meant by "you can leave the first two lines as is", I think the mock stuff will look less readable if we attempt to split it. My comment was aimed at the other two lines which seemed to be able to fit if doing something like

```python
    self.assertEqual(
        GoogleKubernetesEngineTest.MOCK_GKE_CLUSTER_OBJECT,
        get_results)
```

and

```python
    get_results = GoogleKubernetesEngineTest.FAKE_GKE.GetCluster(
        'projects/fake-project/locations/fake-region/clusters/fake-cluster')
```